pkgs/applications/networking/cluster/helm/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchurl, kubernetes }:
+let
+  arch = if stdenv.isLinux
```

Comment:
I think it a build from source would be possible. I could have a go at doing that.